### PR TITLE
Deleting and recreating gh-pages docs and examples for 1.1

### PR DESCRIPTION
**Pull request by nikhiljindal**
1. rm -rf _v1.1/docs/ _v1.1/examples/
2. Fetching updated docs using `go run _tools/release_docs/main.go _tools/release_docs/api-reference-process.go --branch release-1.1 --output-dir _v1.1 --version v1.1`

cc @bgrant0607

### CODE REVIEW

**Comment by bgrant0607** [comment]
Thanks. Search for "/HEAD/". Why did those 46 links get un-versionized?

**Comment by bgrant0607** [comment]
Looks like this fixed the redundant titles.

File: _v1.1/docs/user-guide/configuring-containers.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ pods/hello-world
 
 `kubectl create --validate` currently warns about problems it detects, but creates the resource anyway, unless a required field is absent or a field value is invalid. Unknown API fields are ignored, so be careful. This pod was created, but with no `command`, which is an optional field, since the image may specify an `Entrypoint`.
 View the [Pod API
-object](https://htmlpreview.github.io/?https://github.com/kubernetes/kubernetes/HEAD/docs/api-reference/v1/definitions.html#_v1_pod)
+object](http://kubernetes.io/v1.1/v1.1.0/docs/api-reference/definitions.html#_v1_pod)
```
Comment (nikhiljindal) [inline _v1.1/docs/user-guide/configuring-containers.md]: This looks like a bug.
cc @caesarxuchao who recently changed this logic.

**Comment by nikhiljindal** [comment]
Yes this fixed the redundant titles and raw, endraw issues.
I can see 2 issues remaining:
* Incorrect munger for htmlpreview links.
* munger linking to HEAD for files that are not in gh-pages: Looks like the script unversions all files that are not included in k8s.io: https://github.com/kubernetes/kubernetes/blob/gh-pages/_tools/release_docs/main.go#L66

@caesarxuchao can look at the first issue and I can look at the second one.

File: _v1.1/docs/user-guide/configuring-containers.md (hunk 1 above)
Comment (bgrant0607) [inline _v1.1/docs/user-guide/configuring-containers.md]: cc @krousey also

Comment (krousey) [inline _v1.1/docs/user-guide/configuring-containers.md]: #13149 for context I think... back to build copping

**Comment by caesarxuchao** [comment]
I'm taking a look now.

**Comment by bgrant0607** [comment]
Merging this one to get a clean starting point.

File: _v1.1/docs/user-guide/configuring-containers.md (hunk 1 above)
Comment (caesarxuchao) [inline _v1.1/docs/user-guide/configuring-containers.md]: fix #16522 will fix this.

**Pull request merged**